uploader/models.py

```diff
@@ -1,3 +1,32 @@
 from django.db import models
 
-# Create your models here.
+
+class SingletonModel(models.Model):
```

Comment:
Why do we need this btw ?

Reply:
Since I was using a single numeric counter so was using a class (model) which would only have one instance. Will be changing this approach completely.